user/tests/wiring/no_fixture_cellular/cellular.cpp

```diff
@@ -202,28 +202,60 @@ test(CELLULAR_06_on_off_validity_check) {
 
     ret = Cellular.command("AT\r\n");
     assertEqual(ret, (int)RESP_OK);
-
-    connect_to_cloud(HAL_PLATFORM_MAX_CLOUD_CONNECT_TIME);
 }
 
 test(CELLULAR_07_urcs) {
-    connect_to_cloud(HAL_PLATFORM_MAX_CLOUD_CONNECT_TIME);
 
 #if HAL_PLATFORM_GEN == 3
     assertEqual(Cellular.command("AT\r\n"), (int)RESP_OK);
+    int modem_type = -1;
+#if HAL_PLATFORM_NCP
+    modem_type = cellular_modem_type();
+#endif // HAL_PLATFORM_NCP
+
+    // Ensure we are disconnected, bad things can happen if messages are tx/rx
```

Comment:
Any specific issues you are seeing? It may interfere with vitals or system describe, but normally does work out fine.

Reply:
It was actually some other command I was running after the AT interface was suspended, but it made me think that other things can happen at just the wrong time causing much longer testing delays due to the modem trying to power cycle at that test (if it was trying to send AT commands while suspended).  So the previous test ends with essentially power cycling the modem, so we really don't need to reconnect right away.  Best to leave cellular just powered on, and do the URC test... then connect to the cloud on the way out of that test.